### DISCUSSION TITLE
Зимняя карта.

### DISCUSSION THREAD

**Discussion by Asdasdasdasdsaddczx2** [octothorp-team/public-dobrograd#755] [Предложения]
На дворе скоро будет Декабрь, разве никто не хочет поотыгрывать разные штуки на зимней карте? Покататься на катке, да и темболее зима красивее атмосферой, своим холодом, предложение сделать зимнюю карту ну или же, заняться созданием новой зимней карты.

**Comment by delingar** [comment]
Карта имеется и разрабатывается для новой версии. Под вопросом только цдбг и его зимний риверден

**Comment by Hartsig** [comment]
Кстати можно добавить чтоб машины немного скользили как по снежной дороге и в принципе можно сделать места для любителей дрифта что бы покатать бочком по снежной дороге

**Comment by VaR1aN** [comment]
Ты видимо не дано начал играть на ДБГ, карта меняется в декабре, какого числа не известно, но зимняя карта всегда под новый год уже есть. Только вопрос с ЦДБГ, так как он появился совсем недавно, зимняя карта может еще не готова.
